modules/persistent-storage-vsphere-static-provisioning.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-vsphere.adoc
+
+[id="vsphere-static-provisioning_{context}"]
+= Statically provisioning VMware vSphere volumes
+
+To statically provision VMware vSphere volumes you must create the virtual machine disks for reference by the persistent volume framework.
+
+.Prerequisites
+
+* Storage must exist in the underlying infrastructure before it can be mounted as a volume in {product-title}.
+
+== Create the virtual machine disks
+
+Virtual machine disks (VMDKs) must be created manually before statically provisioning VMware vSphere volumes. Use either of the following methods:
```

Comment:
I'd change the nesting so this line was the step and the next two options were nested under it.